Source/AwsCommonRuntimeKit/io/TlsConnectionOptions.swift

```diff
@@ -5,35 +5,35 @@ import AwsCIo
 
 public final class TlsConnectionOptions {
 	private let allocator: Allocator
-    var rawValue: UnsafeMutablePointer<aws_tls_connection_options>
-
+    private var _rawValue: UnsafeMutablePointer<aws_tls_connection_options>
+	var rawValue: UnsafePointer<aws_tls_connection_options> { UnsafePointer(_rawValue) }
 	init(_ context: TlsContext, allocator: Allocator) {
 		self.allocator = allocator
-        self.rawValue = allocator.allocate(capacity: 1)
-		aws_tls_connection_options_init_from_ctx(rawValue, context.rawValue)
+        self._rawValue = allocator.allocate(capacity: 1)
+		aws_tls_connection_options_init_from_ctx(_rawValue, context.rawValue)
         #if os(iOS) || os(watchOS)
-		rawValue.pointee.timeout_ms = 30_000
+		_rawValue.pointee.timeout_ms = 30_000
         #else
-		rawValue.pointee.timeout_ms = 3_000
+		_rawValue.pointee.timeout_ms = 3_000
         #endif
     }
 
 	public func setAlpnList(_ alpnList: String) throws {
-		if aws_tls_connection_options_set_alpn_list(rawValue, self.allocator.rawValue, alpnList) != AWS_OP_SUCCESS {
+		if aws_tls_connection_options_set_alpn_list(_rawValue, self.allocator.rawValue, alpnList) != AWS_OP_SUCCESS {
```

Comment:
like I said above, this function can't *really* fail, it's from the bad old days where any allocation could fail

Reply:
Thanks, fixed.